src/h3lib/lib/bbox.c

```diff
@@ -87,39 +87,28 @@ double _hexRadiusKm(H3Index h3Index) {
     return _geoDistKm(&h3Center, h3Boundary.verts);
 }
 
-/**
- * Get the radius of the bbox in hexagons - i.e. the radius of a k-ring centered
- * on the bbox center and covering the entire bbox.
- * @param  bbox Bounding box to measure
- * @param  res  Resolution of hexagons to use in measurement
- * @return      Radius in hexagons
- */
-int bboxHexRadius(const BBox* bbox, int res) {
-    // Determine the center of the bounding box
-    GeoCoord center;
-    bboxCenter(bbox, &center);
-
-    // Use a vertex on the side closest to the equator, to ensure the longest
-    // radius in cases with significant distortion. East/west is arbitrary.
-    double lat =
-        fabs(bbox->north) > fabs(bbox->south) ? bbox->south : bbox->north;
-    GeoCoord vertex = {lat, bbox->east};
-
-    // Determine the length of the bounding box "radius" to then use
-    // as a circle on the earth that the k-rings must be greater than
-    double bboxRadiusKm = _geoDistKm(&center, &vertex);
+int bboxHexEstimate(const BBox* bbox, int res) {
+    // Get the area of the pentagon as the maximally-distored area possible
+    H3Index pentagons[12] = {0};
+    H3_EXPORT(getPentagonIndexes)(res, pentagons);
+    double pentagonRadiusKm = _hexRadiusKm(pentagons[0]);
+    // Area of a regular hexagon is 3/2*sqrt(3) * r * r
```

Comment:
This comment is confusing when you're actually finding pentagon area. It also seems like this isn't really the pentagon area at all?

I think this is OK for now, but if the intent is "Use the area of the smallest cell at this res for the calculation", I'd feel a lot better if we actually had precalculated stats in the library, as we've been discussing, and just used that.

Reply:
That's true, I'm calculating the area of a hexagon using the edge length of one of the pentagons. It's confusing but it does work, since that very same edge length exactly matches the edge length of the hexagons in the first ring around the pentagon and is therefore the minimum hexagon size.

I can add some explanatory comments on this weirdness to make all of that more clear.